recipe/meta.yaml

```
@@ -11,7 +11,7 @@ source:
 
 build:
   noarch: python
-  number: 1
+  number: 0
   script: python -m pip install --no-deps .
```

This is what we are recommending these days. `--no-deps` is handled by `conda-build`. The extra verbosity should help us pin point any install errors that may be happening.

```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```